FamiliyDoctor/FamiliyDoctor/src/main/java/com/familydoctor/models/DTO/MedicalRecordDTO.java

```diff
@@ -0,0 +1,32 @@
+package com.familydoctor.models.DTO;
+
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class MedicalRecordDTO {
+    @NotBlank
+    private Long id;
+    @NotBlank
+    private String patientFirstName;
```

Comment:
If you already have a Patient object/entity, do you need to repeat the patient's properties in this class? Can't we use the Patient object in this MedicalRecord, as a property?